### PR TITLE
Release assets to s3

### PR DESCRIPTION
**Pull request by j00k**

### CODE REVIEW

**Comment by eddywashere** [comment]
woot! do you have a url for this that we can test and verify headers?

File: script/release [hunk 1]
```diff
@@ -33,3 +33,5 @@ rack files object upload-dir \
   --dir $PACKAGE_ROOT \
   --region IAD \
   --recurse
+
+aws s3 sync $PACKAGE_ROOT s3://canon.rackspace.com --region us-west-2
```
Comment (j00k) [inline script/release]: tbh, i'm not sure.

from what i can tell we'll end up with

cloudfront.blah/package/$CHANNEL/$VERSION

**Comment by j00k** [comment]
Not yet, I'm going to tweak cache settings once I verify that releases are pushed properly.

**Comment by eddywashere** [comment]
Sounds good to me. :+1:

**Pull request merged**